@xen-orchestra/lite/src/components/pool/dashboard/AlarmRow.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <tr v-if="isMobile" class="mobile" @click="handleCollapse">
+    <td v-if="isMobile" class="description" colspan="5">
```

Comment:
```suggestion
    <td class="description" colspan="5">
```

Unneeded `v-if` since it's already present on the parent.